src/phoenix/server/app.py

```diff
@@ -57,7 +57,7 @@ async def dispatch(
         return response
 
 
-class GraphQLWithContext(GraphQL):
+class GraphQLWithContext(GraphQL):  # type: ignore
```

Comment:
oooh what happened here?

Reply:
mypy thinks it's a generic in need of parameterization, but i couldn't figure out how it got to that conclusion